python/cairo-addons/src/cairo_addons/hints/hashdict.py

```diff
@@ -77,7 +77,7 @@ def get_preimage_for_key(
 ):
     from cairo_addons.hints.hashdict import _get_preimage_for_hashed_key
 
-    preimage = bytes(
+    preimage = list(
```

Comment:
why ?
What changed for this to be necessary?

Reply:
the preimage is not necessarily containing bytes - we just want a collection of values. for a Bytes32 the preimage is made of two 16-bytes values